utils/routes/home/routes.tsx

```diff
@@ -9,7 +9,7 @@ export type Route = {
 
 export const routes: Array<Route> = [
   {
-    name: 'Home',
+    name: 'MainFeed',
```

Comment:
Is the plan to call it MainFeed? If so the name needs to be changed in the home/names and home/paramList

Reply:
Oh I actually don't know why I changed this. Must have been on accident. Reverting now! Thanks for catching.